File: examples/java-jfr/rideshare/src/main/java/org/example/rideshare/car/CarService.java
package org.example.rideshare.car;

import org.example.rideshare.OrderService;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Service;

@Service
public class CarService {
    @Autowired
    OrderService orderService;

    public void orderCar(int searchRadius) {
        orderService.findNearestVehicle(searchRadius, "car");
    }

}

File: examples/language-sdk-instrumentation/dotnet/rideshare-opentracing/example/CarService.cs
namespace Example;

internal class CarService
{
    private readonly OrderService _orderService;

    public CarService(OrderService orderService)
    {
        _orderService = orderService;
    }

    public void Order(int searchRadius)
    {
        _orderService.FindNearestVehicle(searchRadius, "car");
    }
}